okta/help.md

```diff
@@ -654,7 +654,61 @@ Example output:
 
 ### Triggers
 
-_This plugin does not contain any triggers._
+#### Users Added Removed from Group
+
+This trigger takes a list of groups and returns updates when users are added or removed.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|
+|----|----|-------|--------|-----------|----|
+|group_ids|[]string|None|True|A list of group ID's|None|
+|interval|integer|300|True|The time in seconds inbetween checks for changes to the groups users|None|
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|users_added_from_groups|[]trigger_type|False|Users added to a group since the last check|
+|users_removed_from_groups|[]trigger_type|False|Users removed to a group since the last check|
+
+Example output:
+
+```
+{
+	"users_removed_from_groups": [{
```

Comment:
Do you have an example blob for additions?

Reply:
done